XenAdmin/Actions/GUIActions/ExternalPluginAction.cs

```diff
@@ -55,12 +55,12 @@ internal class ExternalPluginAction : AsyncAction
         private const string BlankParamter = "blank";
 
         private static readonly string SnapInTrustedCertXml = 
-            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), 
-            "Citrix\\XenServerPSSnapIn\\XenServer_Known_Certificates.xml");
+            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData),
+            Branding.COMPANY_NAME_SHORT + "\\XenServerPSSnapIn\\XenServer_Known_Certificates.xml");

```

Comment:
I think that we can use Path.Combine with 3 parameters here (and on line 63) 
Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), Branding.COMPANY_NAME_SHORT, "XenServerPSSnapIn\\XenServer_Known_Certificates.xml");